<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiamondDialog</class>
 <widget class="QDialog" name="DiamondDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Diamond info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="indexLabel">
       <property name="text">
        <string>Index</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="indexLine">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="caratLabel">
       <property name="text">
        <string>Carat</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cutLabel">
       <property name="text">
        <string>Cut</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="caratLine"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cutComboBox">
       <item>
        <property name="text">
         <string>Fair</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Good</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very Good</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Premium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ideal</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="colorComboBox">
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="clarytiLabel">
       <property name="text">
        <string>Clarity</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="clarityComboBox">
       <item>
        <property name="text">
         <string>I3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SI2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SI1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VS1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VVS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VVS1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="priceLine"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="depthLine"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="tableLabel">
       <property name="text">
        <string>Table</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="tableLine"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lengthLine"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="widthLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiamondDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiamondDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
